buildSrc/src/main/kotlin/dependency-management.gradle.kts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2022, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import io.spine.examples.kanban.dependency.ApiGuardian
+import io.spine.examples.kanban.dependency.CheckerFramework
+import io.spine.examples.kanban.dependency.Grpc
+import io.spine.examples.kanban.dependency.Guava
+import io.spine.examples.kanban.dependency.JUnit
+import io.spine.examples.kanban.dependency.Truth
+
+plugins {
```

Comment:
As long as perform such a radical extraction of `build.gradle` pieces, let's have a file-level comment in each of them. The goal is to tell about the purpose of each of the "plugins" and, perhaps, some details that worth mentioning. 

The pattern here is as usual. The first paragraph should be just a single sentence (or maybe two, if needed) telling what this is. The following paragraphs expand the topic providing more details.

Remember, this is a part of an _example_. So it's a part of educational process, as well as a working application.